src/client/rekor.ts

```diff
@@ -115,9 +115,9 @@ export interface SearchIndex {
 }
 
 export interface SearchLogQuery {
-  entries?: HashedRekordOptions[];
+  entries?: EntryKind[];
   entryUUIDs?: string[];
-  logIndexes: number[];
+  logIndexes?: number[];
 }
```

Comment:
At some point I guess I had started to add this API myself. Have no memory of this 😆